app/experimenter/legacy-ui/templates/base.html

```diff
@@ -75,6 +75,27 @@ <h3 class="pt-2">
     </div>
   </div>
 
+  <div id="nimbus-cta">
+    <div class="container">
+      <div class="row py-3">
+        <div class="col-9">
+          <h5 class="font-bold">
```

Comment:
H5 is possibly an a11y issue here, but I don't think the page currently passes a11y standards by starting with an H3 anyways (or at least Lighthouse is kvetching about non-sequential headings), so 🤷 .

Reply:
Oooh yeah good point. I'm somewhat less concerned just because we're gonna junk the whole application soon anyhow, but I could spend some time finessing it into a <p> tag or something